src/object/obj_tx.c

```diff
@@ -2787,6 +2809,9 @@ dc_tx_per_akey_existence_task(enum obj_rpc_opc opc, daos_handle_t oh, struct dc_
 			D_MUTEX_UNLOCK(&tx->tx_lock);
 			obj_decref(obj);
 
+			if (tx->tx_for_convert)
+				rc = dc_tx_convert_post(tx, parent, opc, 0);
```

Comment:
[Question] If the dc_tx_add_update call above returns a nonzero rc, should we still call dc_tx_convert_post and override rc?